[CS2113-F15-4] BookBuddy

yeozongyao opened the pull request:
BookBuddy helps users track and manage the list of the books they are reading. It is optimized for CLI users so that the tracking and management objectives can be achieved most efficiently.

Luo-Z-Y commented on docs/DeveloperGuide.md:
```diff
@@ -4,15 +4,110 @@
+from the main code. It is instead replaced by a simple interface for the user to work with, adhering to the abstraction concept of
+object-oriented programming.
 
 ## Product scope
```
![image](https://github.com/nus-cs2113-AY2324S2/tp/assets/97681546/77fdc04e-d54f-4478-9f4b-1b2d60f4f4a5)
Could you give more elaborations on user stories?

JingHaoooo commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,194 @@
+- An embedded approach, where genre setting logic is part of a larger class managing all book attributes, was 
+  considered. However, this was rejected due to potential scalability issues and difficulty in maintaining code.
+
+![SetGenreSequenceDiagram.png](UML_diagrams/SetGenreSequenceDiagram.png)
```
![image](https://github.com/nus-cs2113-AY2324S2/tp/assets/99406560/dba9a645-b442-48c2-9603-bd4a7657c6fc)

Is this correct? Should it be returning any value(eg genres)?

JingHaoooo commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,194 @@
+By importing predefined string constants from `CommandList` class representing the valid commands, the `ParserMain` class 
+parses the input from a user using the `parseCommand` method. The class diagram below shows how `ParserMain` interacts with other
+classes.
+![ParserMain.png](UML_diagrams/ParserMain.png)
```
![image](https://github.com/nus-cs2113-AY2324S2/tp/assets/99406560/28e1f0be-6bd4-44a2-aa18-e152fef378f2)
Perhaps you could remove the icons in the class diagrams, as well as the attributes icons.

JingHaoooo commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,194 @@
+| Version | As a ... | I want to ...                                 | So that I can ...                                         |
+|---------|----------|-----------------------------------------------|-----------------------------------------------------------|
+| v1.0    | new user | see usage instructions                        | refer to them when I forget how to use the application    |
+| v1.0    | user     | add books to a list                           |                                                           |
+| v1.0    | user     | remove books from a list                      |                                                           |
```
Perhaps you could add the part for "so that I can"  to make it complete.

JingHaoooo reviewed (commented):
Overall good job. Diagrams were mostly clear and easy to understand.

LowTL commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,194 @@
+
+Reference to AB-3 diagrams code
+* [Source URL](https://github.com/se-edu/addressbook-level3/tree/master/docs/diagrams)
+* Used as reference to understand PlantUML syntax
```
I like how you give credit to the sources you referenced!

LowTL commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,194 @@
+  enhancing maintainability and scalability.
+
+#### Alternatives Considered:
+
```
Good display of alternatives considered!

LowTL commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,194 @@
+
+By abstracting out the parsing functionality of BookBuddy into a separate `ParserMain` class, the complexity of parsing 
+user input is removed from the main code. It is instead replaced by a simple interface for the user to work with, adhering to the abstraction
+concept of object-oriented programming.
```
Good application of OOP concepts!

LowTL commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,194 @@
 
 ### Value proposition
 
 {Describe the value proposition: what problem does it solve?}
```
Perhaps you could add a value proposition so that developers can understand at a glance what your program can offer.

LowTL commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,194 @@
+
 ## Design & implementation
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
```
Perhaps you could remove the placeholder text so that it's clearer where the Design and Implementation section begins

LowTL commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,194 @@
 # Developer Guide
 
+## Table of Contents
```
Good use of a Table of Contents to allow easy navigation of the DG!

LowTL reviewed (commented):
Well done on the DG! There are a few small placeholder texts to delete, but generally well done!

liuzehui03 committed:
Merge pull request #86 from liuzehui03/expanding_find_function

find based on read and unread status implemented

liuzehui03 committed:
updated UG and list of commands in help

liuzehui03 committed:
Merge pull request #87 from liuzehui03/expanding_find_function

updated UG and list of commands in help

yeozongyao committed:
Add single step operation of set-genre method for pro users

yeozongyao committed:
Checkstyle fixes

yeozongyao committed:
Checkstyle fixes

yeozongyao committed:
Merge pull request #88 from yeozongyao/branch-zongyao-setgenre

Add single step operation of set-genre method for pro users

yeozongyao committed:
Update user guide

yeozongyao committed:
Merge pull request #89 from yeozongyao/branch-zongyao-setgenre

Update user guide

yeozongyao committed:
Update developer guide

yeozongyao committed:
Authorship edits

yeozongyao committed:
Merge pull request #90 from yeozongyao/branch-zongyao-setgenre

Update developer guide

liuzehui03 committed:
fix bug such that capitalization of letters do not matter when using find-title and find-genre

lordgareth10 committed:
Update UG

liuzehui03 committed:
fix checkstyle error

lordgareth10 committed:
Merge pull request #95 from lordgareth10/master

Update UG

liuzehui03 committed:
Merge pull request #94 from liuzehui03/master

fix bug such that capitalization of letters do not matter when using …

yeozongyao committed:
Bug fix for set-genre feature

yeozongyao committed:
Update Set-Genre junit test

yeozongyao committed:
Checkstyle fixes

yeozongyao committed:
Checkstyle fixes

yeozongyao committed:
Merge pull request #96 from yeozongyao/branch-zongyao-setgenre

Bug fix for set-genre feature

lordgareth10 committed:
Fix bugs in genre and label commands

lordgareth10 committed:
Merge pull request #97 from lordgareth10/master

Fix bugs in genre and label commands

liuzehui03 committed:
update UG

liuzehui03 committed:
Merge pull request #98 from liuzehui03/master

update UG

liuzehui03 committed:
no message

liuzehui03 committed:
Merge pull request #99 from liuzehui03/master

no message

yeozongyao committed:
Authorship edits

yeozongyao committed:
Merge pull request #100 from yeozongyao/branch-zongyao-setgenre

Authorship edits

liuzehui03 committed:
find_label feature

liuzehui03 committed:
fixed error

liuzehui03 committed:
Merge pull request #101 from liuzehui03/master

find_label feature

liuzehui03 committed:
update find-label feature

liuzehui03 committed:
Merge pull request #102 from liuzehui03/master

update find-label feature

liuzehui03 committed:
created separate class for BookFind

liuzehui03 committed:
fix checkstyle error

liuzehui03 committed:
fix checkstyle

liuzehui03 committed:
Merge pull request #103 from liuzehui03/master

created separate class for BookFind

Joshuahoky committed:
Add exception handling for FileStorage to deal with corrupted text file

Joshuahoky committed:
Fix checkstyle issue

Joshuahoky committed:
Merge pull request #104 from Joshuahoky/branch-fileStorage

Add exception handling for FileStorage to deal with corrupted text file

Joshuahoky committed:
Add more exception handling for corrupted text file

Joshuahoky committed:
Merge pull request #105 from Joshuahoky/branch-fileStorage

Add more exception handling for corrupted text file

lordgareth10 committed:
Add datetime read feature

lordgareth10 committed:
Merge branch 'master' of https://github.com/AY2324S2-CS2113-F15-4/tp into Datetime

# Conflicts:
#	src/main/java/seedu/bookbuddy/FileStorage.java
#	src/main/java/seedu/bookbuddy/book/BookMain.java
#	src/main/java/seedu/bookbuddy/booklist/BookListModifier.java

lordgareth10 committed:
Fix checkstyle

lordgareth10 committed:
Merge pull request #146 from lordgareth10/Datetime

Add Datetime read to book

liuzehui03 committed:
updat BookFind

liuzehui03 committed:
no message

liuzehui03 committed:
no message

liuzehui03 committed:
checkstyle fixes

liuzehui03 committed:
checkstyle fixes

liuzehui03 committed:
Merge pull request #147 from liuzehui03/master

updat BookFind

liuzehui03 committed:
update Ui

liuzehui03 committed:
Merge pull request #148 from liuzehui03/master

update Ui

liuzehui03 committed:
improve ui

liuzehui03 committed:
added assertions

liuzehui03 committed:
junit test fixes

liuzehui03 committed:
Merge pull request #149 from liuzehui03/master

modifications

liuzehui03 committed:
fix find-genre issue

liuzehui03 committed:
Merge pull request #150 from liuzehui03/master

fix find-genre issue

liuzehui03 committed:
added find-rate

liuzehui03 committed:
checkstyle fix

liuzehui03 committed:
checkstyle fix

liuzehui03 committed:
Merge pull request #151 from liuzehui03/master

find-rate function added

liuzehui03 committed:
update UG

liuzehui03 committed:
Merge pull request #152 from liuzehui03/master

update UG

lordgareth10 committed:
Add list-by-date command

lordgareth10 committed:
Merge pull request #153 from lordgareth10/master

Add list-by-date command

lordgareth10 committed:
Update UG and add javadocs

lordgareth10 committed:
Merge pull request #154 from lordgareth10/master

Update UG and add javadocs

yeozongyao committed:
Bug fixes -
1. Set-genre does not take into trailling and leading spaces
2. Set-genre does not detect existing genres in the genre list

yeozongyao committed:
Checkstyle fix - wrong indentation for javadoc

yeozongyao committed:
Checkstyle fixes for gareth - wrong indentation for javadoc

yeozongyao committed:
Checkstyle fix for gareth - wrong indentation for javadoc

yeozongyao committed:
Checkstyle fixes for gareth - wrong indentation for javadoc

yeozongyao committed:
Merge pull request #155 from yeozongyao/branch-zongyao-pedbugs

Bug fixes -

yeozongyao committed:
Bug fixes -
1. [PE-D][Tester C] Usablity issues caused by not being able to end the program using METHOD: bye when using METHOD: set-genre
2. [PE-D][Tester A] Set Genre Command not Taking Other Numbers into Account

Enhanced exception handling and parsing process to ensure invalid inputs will not be processed.
Extracted exit process in main class and created a public wrapper to perform the exit anywhere in the programme

yeozongyao committed:
Checkstyle fixes - line longer than 120 characters

yeozongyao committed:
Merge pull request #156 from yeozongyao/branch-zongyao-pedbugs

Bug fixes -

yeozongyao committed:
Test

yeozongyao committed:
Bug fixes -
1. [PE-D][Tester F] Inproper Exception Handling #133
2. [PE-D][Tester B] Redundant output in the label command #112

Enhanced exception handling to capture the different wrong input scenarios

yeozongyao committed:
Checkstyle fixes - lines longer than 120 characters

yeozongyao committed:
Checkstyle fixes - wrong indentation

yeozongyao committed:
Merge pull request #158 from yeozongyao/branch-zongyao-pedbugs

Branch zongyao pedbugs

yeozongyao committed:
Bug fixes -
1. [PE-D][Tester A] Documentation Giving Misleading Instructions #144
2. [PE-D][Tester B] The user-input genre type can't be saved #123

Changed the user guide to be more specific and enhanced file storage to save the available genre list

yeozongyao committed:
Checkstyle fixes - wrong variable declaration order

yeozongyao committed:
Merge pull request #159 from yeozongyao/branch-zongyao-pedbugs

Bug fixes -

yeozongyao committed:
Bug fix -

DG Review By TA (Jai): Incorrect Notation In Sequence Diagram #108
Missing colons in the instance name

yeozongyao committed:
Bug fix -
DG Review By TA (Jai): Incorrect Notation In Sequence Diagram - 2 #110
Name of instances mentioned at the bottom also

yeozongyao committed:
Bug fixes -
DG Review By TA (Jai): Missing Activation Bar In Sequence Diagram #109

Added the activation bars

yeozongyao committed:
Merge pull request #160 from yeozongyao/branch-zongyao-pedbugs

Branch zongyao pedbugs

yeozongyao committed:
Update Developer Guide to include sequence diagrams for set-label feature.

yeozongyao committed:
Merge pull request #161 from yeozongyao/branch-zongyao-pedbugs

Update Developer Guide to include sequence diagrams for set-label fea…

lordgareth10 committed:
Documentation and refactoring

lordgareth10 committed:
Merge pull request #162 from lordgareth10/master

Documentation and refactoring

lordgareth10 committed:
Refactor printbookdbydateread method

lordgareth10 committed:
Merge branch 'master' of https://github.com/AY2324S2-CS2113-F15-4/tp

yeozongyao committed:
Bug fix - immutable list cannot be modified. Changed it to be modifiable

yeozongyao committed:
Merge pull request #164 from yeozongyao/branch-zongyao-pedbugs

Bug fix - immutable list cannot be modified. Changed it to be modifiable

lordgareth10 committed:
Merge branch 'master' of https://github.com/AY2324S2-CS2113-F15-4/tp

lordgareth10 committed:
Bug FIx: Mishandling for incorrect user commands for METHOD: list

lordgareth10 committed:
FIx bugs for all list commands

lordgareth10 committed:
Merge pull request #165 from lordgareth10/master

Bug Fix PED: Mishandling for incorrect user commands for METHOD: list

Joshuahoky committed:
Update User Guide for improved readability

Joshuahoky committed:
Merge pull request #166 from Joshuahoky/master

Update User Guide for improved readability

Joshuahoky committed:
Fix bug regarding message displayed when removing book from empty list

Joshuahoky committed:
Fix checkstyle

Joshuahoky committed:
Merge pull request #167 from Joshuahoky/branch-bugFixing

Fix bug regarding message displayed when removing book from empty list

lordgareth10 committed:
Add author for book

lordgareth10 committed:
Fix checkstyle

lordgareth10 committed:
Merge pull request #168 from lordgareth10/branch-Author

Branch author

lordgareth10 committed:
Add check for duplicate books for add and set-author commands

lordgareth10 committed:
Merge pull request #169 from lordgareth10/master

Add check for duplicate books for add and set-author commands

yeozongyao committed:
Added PPP for zong yao

yeozongyao committed:
Update PPP

yeozongyao committed:
Merge pull request #170 from yeozongyao/branch-zongyao-pedbugs

Added PPP for zong yao

lordgareth10 committed:
Update UG and help command

liuzehui03 committed:
Update PPP

liuzehui03 committed:
Merge pull request #172 from liuzehui03/master

Update PPP

liuzehui03 committed:
added find-author

liuzehui03 committed:
Added comments

liuzehui03 committed:
Checkstyle fixes

liuzehui03 committed:
Merge pull request #173 from liuzehui03/master

Added find-author and comments

lordgareth10 committed:
Merge pull request #171 from lordgareth10/master

Update UG and help command

lordgareth10 committed:
PPP update

lordgareth10 committed:
Merge pull request #174 from lordgareth10/master

PPP update

lordgareth10 committed:
Bug Fix: [PE-D][Tester E] Adding a space before the index makes the command invalid #136

lordgareth10 committed:
Bug Fix: [PE-D][Tester D] Function accepts commands with extra parameters #122

lordgareth10 committed:
Bug Fix: [PE-D][Tester F] Inproper Exception Handling #135

lordgareth10 committed:
Merge pull request #175 from lordgareth10/master

Bug Fixes

Joshuahoky committed:
Update PPP

Joshuahoky committed:
Merge pull request #176 from Joshuahoky/master

Update PPP

Joshuahoky committed:
Add more exception handling for fileStorage class

Joshuahoky committed:
Update UG

Joshuahoky committed:
Merge pull request #177 from Joshuahoky/branch-bugFixing

Branch bug fixing

Joshuahoky committed:
Update DG

Joshuahoky committed:
Merge pull request #178 from Joshuahoky/master

Update DG

yeozongyao committed:
Clean up code for set-genre, label and rate features and minor code enhancements for list, list-rated and add feature

yeozongyao committed:
Merge pull request #179 from yeozongyao/branch-zongyao-codeenhancements

Clean up code for set-genre, label and rate features and minor code e…

yeozongyao committed:
Added Junit tests for SetBookRating, ParserRating, ParseList, ParseListRated

yeozongyao committed:
Fix system support error for JUnit tests

yeozongyao committed:
Merge pull request #180 from yeozongyao/branch-zongyao-testcodes

Added Junit tests for SetBookRating, ParserRating, ParseList, ParseLi…

liuzehui03 committed:
Fix issue #118

liuzehui03 committed:
Merge pull request #181 from liuzehui03/master

Fix issue #118

yeozongyao committed:
Bug fix - change availablegenre arraylist to be non-static. Reformatted for the rest of the code base

yeozongyao committed:
Checkstyle fix - wrong indentation

yeozongyao committed:
Merge pull request #182 from yeozongyao/branch-zongyao-testcodes

Bug fix - change availablegenre arraylist to be non-static. Reformatt…

Joshuahoky committed:
Fix minor bugs and add testing for remove and display commands

Joshuahoky committed:
Merge pull request #188 from Joshuahoky/branch-testing

Fix minor bugs and add testing for remove and display commands

yeozongyao committed:
Fixed bug where user is unable to exit from secondary input to primary input in set-genre function and enhanced the code by including indentations for the respective level of input

yeozongyao committed:
Checkstyle fix - line longer than 120 lines

yeozongyao committed:
Merge pull request #189 from yeozongyao/branch-zongyao-testcodes

Fixed bug where user is unable to exit from secondary input to primary input in set-genre function

yeozongyao committed:
Fix bug where user will not get warning if giving same rating or label to a book when they have already given the same one before

yeozongyao committed:
Merge pull request #190 from yeozongyao/branch-zongyao-testcodes

Fix bug where user will not get warning if giving same rating or label to a book when they have already given the same one before

yeozongyao committed:
Added features to remove genre and to list the current available genres

yeozongyao committed:
Checkstyle fixes - end with new line

yeozongyao committed:
Merge pull request #191 from yeozongyao/branch-zongyao-testcodes

Added features to remove genre and to list the current available genres

yeozongyao committed:
Add Junit test cases for set-genre, rate, label, add, list, list-genre, list-rated and remove-genre features

yeozongyao committed:
Checkstyle fixes

yeozongyao committed:
Merge pull request #192 from yeozongyao/branch-zongyao-testcodes

Add Junit test cases for features by zong yao

yeozongyao committed:
Bug fix - Exception message should be more accurate

yeozongyao committed:
Added documentation for features under yeozongyao

yeozongyao committed:
Checkstyle fixes

yeozongyao committed:
Merge pull request #195 from yeozongyao/branch-zongyao-codeenhancements

Added documentation and minor bug fixes

yeozongyao committed:
Added documentation in developer guide - Set Rating feature and BookMain Class

yeozongyao committed:
Merge pull request #196 from yeozongyao/branch-zongyao-codeenhancements

Added documentation in developer guide - Set Rating feature and BookM…

yeozongyao committed:
Bug fix - set-genre feature loop does not continue when a wrong input is entered

yeozongyao committed:
Merge pull request #197 from yeozongyao/branch-zongyao-codeenhancements

Bug fix - set-genre feature loop does not continue when a wrong input…

liuzehui03 committed:
Update UG and DG

liuzehui03 committed:
Merge pull request #198 from liuzehui03/master

Update UG and DG

liuzehui03 committed:
Update Ui help message

liuzehui03 committed:
Merge pull request #199 from liuzehui03/master

Update Ui help message

Joshuahoky committed:
Updae DG, UG and PPP

Joshuahoky committed:
Merge pull request #200 from Joshuahoky/master

Updae DG, UG and PPP

Joshuahoky committed:
Update UG and PPP

Joshuahoky committed:
Merge pull request #201 from Joshuahoky/master

Update UG and PPP

liuzehui03 committed:
Update DG

liuzehui03 committed:
Merge pull request #202 from liuzehui03/master

Update DG

liuzehui03 committed:
Update DG

liuzehui03 committed:
Merge pull request #203 from liuzehui03/master

Update DG

lordgareth10 committed:
Add more JUnit tests for find methods and exceptions

yeozongyao committed:
Bug fix - added accurate exception messages

yeozongyao committed:
Merge branch 'master' of https://github.com/AY2324S2-CS2113-F15-4/tp into branch-zongyao-codeenhancements

* 'master' of https://github.com/AY2324S2-CS2113-F15-4/tp:
  Update DG
  Update DG
  Update UG and PPP
  Updae DG, UG and PPP
  Update Ui help message
  Update UG and DG

yeozongyao committed:
Merge pull request #204 from yeozongyao/branch-zongyao-codeenhancements

Bug fix - accurate exception messages

lordgareth10 committed:
Add UML diagrams

lordgareth10 committed:
Merge pull request #205 from lordgareth10/master

Add UML diagrams and more JUnit tests

liuzehui03 committed:
Update PPP DG and junit test

liuzehui03 committed:
Update JUNIT

liuzehui03 committed:
Fix merge fail issue

liuzehui03 committed:
Merge pull request #206 from liuzehui03/master

Update PPP DG and junit test

liuzehui03 committed:
Added junit test for find-read and find-unread

liuzehui03 committed:
Merge pull request #207 from liuzehui03/master

Added junit test for find-read and find-unread

liuzehui03 committed:
Added JUNIT for find genre

liuzehui03 committed:
Merge pull request #208 from liuzehui03/master

Added JUNIT for find genre

Joshuahoky committed:
Update UG

Joshuahoky committed:
Merge branch 'master' of https://github.com/AY2324S2-CS2113-F15-4/tp

yeozongyao committed:
Bug fix - rate 0 1 and label 0 1 will give a wrong error message

Joshuahoky committed:
Update UG and Ui help message

yeozongyao committed:
Bug fix - testInvalidIndexOutofRange test fail

Joshuahoky committed:
Merge pull request #211 from Joshuahoky/master

Update UG

yeozongyao committed:
Update help menu

yeozongyao committed:
Merge branch 'master' of https://github.com/AY2324S2-CS2113-F15-4/tp into branch-zongyao-codeenhancements

* 'master' of https://github.com/AY2324S2-CS2113-F15-4/tp:
  Update UG and Ui help message
  Update UG

# Conflicts:
#	src/main/java/seedu/bookbuddy/Ui.java

yeozongyao committed:
Merge pull request #209 from yeozongyao/branch-zongyao-codeenhancements

Bug fix - rate 0 1 and label 0 1 will give a wrong error message

liuzehui03 committed:
Update PPP

yeozongyao committed:
Update PPP for zongyao

liuzehui03 committed:
Update PPP

liuzehui03 committed:
Merge pull request #212 from liuzehui03/master

Update PPP

yeozongyao committed:
Update PP, About Us and read me

yeozongyao committed:
Merge pull request #213 from yeozongyao/branch-zongyao-codeenhancements

Branch zongyao codeenhancements

lordgareth10 committed:
Bug Fixes: Corrected error messages

liuzehui03 committed:
Update DG

liuzehui03 committed:
Merge pull request #215 from liuzehui03/master

Update DG

lordgareth10 committed:
Update PPP

liuzehui03 committed:
Update PPP

lordgareth10 committed:
Merge pull request #214 from lordgareth10/master

Bug Fixes: Corrected error messages

liuzehui03 committed:
Merge pull request #216 from liuzehui03/master

Update PPP

Joshuahoky committed:
Update PPP

Joshuahoky committed:
Merge pull request #217 from Joshuahoky/master

Update PPP

yeozongyao committed:
Bug fix - replace split criteria as \\s+ instead of space

liuzehui03 committed:
Update PPP

liuzehui03 committed:
Merge pull request #219 from liuzehui03/master

Update PPP

yeozongyao committed:
Merge pull request #218 from yeozongyao/branch-zongyao-codeenhancements

Bug fix - replace split criteria as \\s+ instead of space

Joshuahoky committed:
Update UG

lordgareth10 committed:
Bug Fixes: Change error message and refactor help command

yeozongyao committed:
Refactor code for set-genre feature for code readability

Joshuahoky committed:
Merge pull request #220 from Joshuahoky/master

Update UG

yeozongyao committed:
Merge pull request #221 from yeozongyao/branch-zongyao-codeenhancements

Refactor code for set-genre feature for code readability

lordgareth10 committed:
Update DG

Joshuahoky committed:
Update UG

lordgareth10 committed:
Fix error messages

Joshuahoky committed:
Merge pull request #222 from Joshuahoky/master

Update UG

lordgareth10 committed:
Checkstyle fixes

lordgareth10 committed:
Merge pull request #223 from lordgareth10/master

Bug Fixes and refactoring

yeozongyao committed:
Update UserGuide.md

liuzehui03 committed:
Update DG

liuzehui03 committed:
Merge pull request #224 from liuzehui03/master

Update DG

yeozongyao committed:
Update AboutUs.md

yeozongyao committed:
Update yeozongyao.md

yeozongyao committed:
Update About Us

liuzehui03 committed:
Update About us

yeozongyao committed:
Merge pull request #225 from yeozongyao/branch-zongyao-codeenhancements

Update About Us

yeozongyao committed:
Update README.md

liuzehui03 committed:
Merge branch 'master' of https://github.com/AY2324S2-CS2113-F15-4/tp

# Conflicts:
#	docs/AboutUs.md

yeozongyao committed:
Update PPP

liuzehui03 committed:
Update DG

liuzehui03 committed:
Merge pull request #226 from liuzehui03/master

Update DG

yeozongyao committed:
Update user guide

yeozongyao committed:
Merge pull request #227 from yeozongyao/branch-zongyao-codeenhancements

Update user guide
